src/CaPTk.cxx

```diff
@@ -109,12 +109,12 @@ int main(int argc, char** argv)
         std::string argv_complete;
         for (size_t i = 2; i < argc; i++) // 2 because the argv[1] is always the "application"
         {
-          argv_complete += " " + std::string(argv[i]);
+          argv_complete += " \"" + std::string(argv[i]) + "\""; // add double quote for command tokenization
```

Comment:
This is probably not going to be enough. The issue is coming from ```CmdParser``` not being able to handle spaces. As a starting point, check this location, where the entire ```argv``` is populated: https://github.com/CBICA/CaPTk/blob/master/src/cbica_toolkit/src/cbicaCmdParser.cpp#L474

Reply:
My bad, I thought the issue was somewhere else.